tests/test_commands.py

```diff
@@ -239,34 +220,6 @@ def setUp(self):
         base = support.BaseCliStub('main', 'updates', 'third_party')
         self.cli = base.mock_cli()
 
-    def test(self):
```

Comment:
Tests were using obscure constructions not easily replicable in SWDB. Repository packages tests were dropped and replaced by functional tests
```
self.cli.base._yumdb.db[str(pkg)] = support.RPMDBAdditionalDataPackageStub()
self.cli.base._yumdb.get_package(pkg).from_repo = 'main'
```
https://github.com/rpm-software-management/dnf/pull/630#issuecomment-278408466
https://github.com/rpm-software-management/ci-dnf-stack/pull/276